README.md

```diff
@@ -766,6 +775,19 @@ Other Style Guides
     function handleThings(opts = {}) {
       // ...
     }
+    
+    // best
+    function isNullOrUndefined(obj){
+       // '==' handles both undefined and null
+       return obj == null;
+    }
+    function getDefaultValueIfNotPresent(obj,defaultValue){
+        return isNullOrUndefined(obj) ? defaultValue : obj;
+    }
+
+    function handleThings(opts = getDefaultValueIfNotPresent(opts,{})){
+       // ...
+    }
```

Comment:
i don't think this adds anything over the existing example of `function handleThings(opts = {}) {`.
```suggestion
```

Reply:
If you explicitly pass `null` to `handleThings` function then `opts` _will be null rather than the default_ `{}` value. Therefore I think we should also handle this case.